contributors/guide/owners.md

```diff
@@ -115,6 +115,24 @@ emeritus_approvers:
 - emily    # 2019-01-05
 ```
 
+#### Cleanup
+
+In addition to the Emeritus process above, from time to time, it is necessary
+to prune inactive members from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. OWNERS have a higher
+activity requirement, as they directly impact the ability of others to contribute.
+If an owner should become inactive, they will be removed but may be added back
+should they become active again.
+
+
+An inactive OWNER is defined as someone with less than 10 Devstats recorded
+contributions within the past year, as shown by this [dashboard]. This is a
```

Comment:
Just need to put a link to the dashboard down below 👍

Reply:
Done. thanks @mrbobbytables